<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SubExposureCalculatorPOC</class>
 <widget class="QMainWindow" name="SubExposureCalculatorPOC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>719</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>719</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SubExposureCalculatorPOC</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="exposureCalculatorMockUp">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>421</width>
      <height>631</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QDoubleSpinBox" name="noiseTolerance">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="allowableNoiseLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>191</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Allowable Noise Increase (%)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QCustomPlot" name="qCustomPlotSubExposure" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>401</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="exposureStackResult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>401</width>
       <height>161</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="SQMLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>41</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>SQM</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="userSkyQuality">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
    </widget>
    <widget class="QTableWidget" name="exposureShotResult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>401</width>
       <height>131</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="filterBandwidth">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>91</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Filter Bandwidth:</string>
     </property>
    </widget>
    <widget class="QFrame" name="gainSelectionNormalFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>284</y>
       <width>401</width>
       <height>30</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QSlider" name="gainSlider">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>6</y>
        <width>341</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximum">
       <number>400</number>
      </property>
      <property name="sliderPosition">
       <number>200</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="gainSliderLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>6</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Gain</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="gainSelectionISODiscreteFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>284</y>
       <width>401</width>
       <height>30</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QComboBox" name="isoDiscreteSelector">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>6</y>
        <width>62</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ISO</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="gainSelectionFixedFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>284</y>
       <width>401</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="gainSelectionFixedLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>6</y>
        <width>381</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Read noise for this camera is uneffected by gain.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="kstarsEkosIndiMockup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>280</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QDoubleSpinBox" name="indiSkyQuality">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>101</width>
       <height>27</height>
      </rect>
     </property>
     <property name="decimals">
      <number>3</number>
     </property>
    </widget>
    <widget class="QLabel" name="indiCameraLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Camera</string>
     </property>
    </widget>
    <widget class="QComboBox" name="imagingSensorSelector">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>241</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="indiSQMLabel">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>SQM</string>
     </property>
    </widget>
    <widget class="QLabel" name="indiFocalRationLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Focal Ratio</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="indiFocalRatio">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>111</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="IndiEkosHeadingLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Initial Data from Indi / EKos</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="applyInitialInput">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>180</y>
       <width>91</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptimal_Exposure_Calclautor">
    <property name="title">
     <string>Optimal Exposure Calculator Proof of Concept</string>
    </property>
   </widget>
   <addaction name="menuOptimal_Exposure_Calclautor"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
